assets/js/editor.js

```diff
@@ -388,6 +394,8 @@
         sbOnUploaded: 'Image téléchargée #image_name#'
       },
       spellChecker: false,
+      inputStyle: 'contenteditable',
+      nativeSpellcheck: 'true',
```

Comment:
```suggestion
      nativeSpellcheck: true,
```

C'est la valeur par défaut donc on n'est pas obligé de le préciser mais comme tu veux. :)

Reply:
c'est la valeur de l'attribut, pas sûr que mettre la valeur en booléen soit adéquate, bien que ça revient au même

Reply:
Ce n'est pas la valeur de l'attribut, c'est la valeur qui est passée à CodeMirror et d'après sa documentation, un booléen est attendu. https://codemirror.net/doc/manual.html#option_spellcheck :)